<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>votacionWin</class>
 <widget class="QWidget" name="votacionWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>951</width>
     <height>621</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/resource/img/elecciones.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lblEleccion">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Elecciones Presidenciales México 2019</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCerrarSesion">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid;
background: #921B63;
color:white;
border-radius: 9px;
font-weight:bold;
font-size:11px;
</string>
   </property>
   <property name="text">
    <string>CERRAR SESIÓN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVotar">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>190</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid #B7387A;
background: #B7387A;
color: white;
border-radius: 15px;
font-weight:bold;
font-size:9px;
</string>
   </property>
   <property name="text">
    <string>VOTAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNombre">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Isaac Ruiz Vázquez</string>
   </property>
  </widget>
  <widget class="QTableView" name="tblCandidatos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>891</width>
     <height>341</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPropuestas">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>190</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #B7387A;
background: none;
color: #B7387A;
border-radius: 15px;
font-weight:bold;
font-size:9px;
</string>
   </property>
   <property name="text">
    <string>VER PROPUESTAS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAtras">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid #B7387A;
background: none;
color: #B7387A;
border-radius: 15px;
font-weight:bold;
font-size:15px;
</string>
   </property>
   <property name="text">
    <string>&lt;  ir atrás</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
